tests/phpunit/includes/bootstrap.php

```diff
@@ -7,8 +7,14 @@
 // load the plugin however *no* Polylang instance is created as no languages exist in DB
 tests_add_filter(
 	'muplugins_loaded',
-	function () use ( $_root_dir ) {
-		require_once $_root_dir . '/polylang.php';
+	function () use ( $_root_dir, $_tests_dir ) {
+		// Kind of doing the job of `wp_register_plugin_realpath()` since the plugin is not in the plugins folder (not even as a symlink).
+		$real_file_path   = realpath( $_root_dir . '/polylang.php' );
+		$symlink_dir_path = wp_normalize_path( dirname( $_tests_dir ) . '/wordpress/wp-content/plugins/polylang' );
+
+		$GLOBALS['wp_plugin_paths'][ $symlink_dir_path ] = wp_normalize_path( dirname( $real_file_path ) );
+
+		require_once $real_file_path;
```

Comment:
Do we need to do the same in all our plugins?

Reply:
Seeing clearer how `plugins_url()` works, I think it's interesting to report this change in Polylang Pro (from `wp-phpunit` package) and Polylang for WooCommerce.

See 
- https://github.com/polylang/wp-phpunit/blob/master/UnitTests/Integration/bootstrap.php#L124-L136
- https://github.com/polylang/polylang-wc/blob/master/tests/phpunit/includes/bootstrap.php#L40 and https://github.com/polylang/polylang-wc/blob/master/tests/phpunit/includes/bootstrap.php#L84-L88


However it seems there is no tests are concerned for the moment.